api/provider/aws/load_balancer_create.go

```diff
@@ -9,14 +9,15 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/aws/aws-sdk-go/service/elb"
+	alb "github.com/aws/aws-sdk-go/service/elbv2"
 	"github.com/aws/aws-sdk-go/service/iam"
 	"github.com/quintilesims/layer0/common/config"
 	"github.com/quintilesims/layer0/common/models"
 )
 
-// Create is used to create an Elastic Load Balancer using the specified Create
-// Load Balancer Request. The Create Load Balancer Request contains the name of
-// the Load Balancer, the Environment ID in which to create the Load Balancer,
+// Create is used to create an Elastic or an Application Load Balancer using the
```

Comment:
We should be consistent with calling our previous implementation of ELBs "classic" load balancers, mostly because it get confusing w `elb` versus `elbv2` and all the other name confusion.

Reply:
✅